packages/openactive-integration-tests/test/features/core/common-error-conditions/implemented/not-found-test.js

```diff
@@ -0,0 +1,49 @@
+const { expect } = require('chai');
+const _ = require('lodash');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+
+/**
+ * Normalize an OpenActive object's @id/id and @type/type fields.
+ *
+ * The response will only have @id and @type fields.
+ *
+ * @param {unknown} oaObject
+ */
+function normalizeAtFields(oaObject) {
+  if (Array.isArray(oaObject)) {
+    return oaObject.map(item => normalizeAtFields(item));
+  }
+  if (_.isPlainObject(oaObject)) {
+    const newObject = {};
+    for (const [key, value] of Object.entries(oaObject)) {
+      const normalizedKey = (key === 'id' || key === 'type')
+        ? `@${key}`
+        : key;
+      const normalizedValue = normalizeAtFields(value);
+      newObject[normalizedKey] = normalizedValue;
+    }
+    return newObject;
+  }
+  return oaObject;
+}
```

Comment:
Of course, this function should be moved out of this file when it's used by other tests. Let me know if this sort of function already existed. I wasn't able to find it

Reply:
Actually the booking spec forces @type - so we should only accept @type and @id within the requests and responses (ref https://github.com/openactive/open-booking-api/issues/141)